### ISSUE TITLE
Shift-space is not a good choice when typing in capitals with a space

### ISSUE THREAD

**Issue by vjpr** [peterjohnhunt/emmet-snippets-compatibility#4]
If you want to write a sentence in capital letters (it happens), the shift-space keybinding causes problems.

E.g. Inside a string or HTML tag. In a comment. Etc.

**Comment by jrial** [comment]
I experience the exact same problem. It's a problematic key binding.

**Comment by GitLiamuk** [comment]
Yeah this is horrible, sticking in content has become such a pain.
